src/lib.rs

```diff
@@ -439,6 +439,28 @@ impl Client {
 
         Ok(entries)
     }
+
+    /// Flushes all existing items on the server
+    ///
+    /// This operation invalidates all existing items immediately. Any items with an update time
+    /// older than the time of the flush_all operation will be ignored for retrieval purposes.
+    /// This operation does not free up memory taken up by the existing items.
+
+    pub async fn flush_all(&mut self) -> Result<(), Error> {
+        self.conn.write_all(b"flush_all\r\n").await?;
+        self.conn.flush().await?;
+
+        let mut response = String::new();
+        self.conn.read_line(&mut response).await?;
+        // check if response is ok
+        if response.trim() == "OK" {
+            Ok(())
+        } else {
+            Err(Error::from(Status::Error(ErrorKind::Protocol(Some(
+                format!("Invalid response for `flush_all` command: `{response}`"),
+            )))))
+        }
```

Comment:
Played around with the existing parser to see if we could leverage that but it looks like it's not built to handle this type of response at the moment.  This works fine for now but it might be better overall to lean in to the parser architecture (potentially in a later PR)?

Reply:
I mostly went with this approach since I saw it on the `version` call. I had a full changeset with `Status::Ok` and all, but it felt a little overkill.